app/post_metadata.go

```diff
@@ -295,7 +295,13 @@ func (a *App) getImagesForPost(post *model.Post, imageURLs []string, isNewPost b
 			imageURLs = append(imageURLs, a.getImagesInMessageAttachments(post)...)
 
 		case model.PostEmbedOpengraph:
-			for _, image := range embed.Data.(*opengraph.OpenGraph).Images {
+			openGraph, ok := embed.Data.(*opengraph.OpenGraph)
+			if !ok {
+				mlog.Debug("Could not read the image data: the data could not be casted to OpenGraph",
```

Comment:
Should we change this to a warning? Although it's not actionable I feel like this is something admins should be aware of. Any thoughts?

Reply:
It depends on how frequent this messages would be, considering that, this is not appeared often. `Warning` level could be fine too.